src/main/java/pl/edukacyjni/user/UserController.java

```diff
@@ -18,4 +19,17 @@ public class UserController {
     public List<User> getAllUsers() {
         return userService.getAllUsers();
     }
+
+    @GetMapping("/{id}")
```

Comment:
That will crash with another PR but that's exactly what I wanted to achieve in this course :D So let's leave it for now and we will discuss it together